orchestra/tests/unit/specs/mistral/test_workflow_spec_validate.py

```diff
@@ -61,14 +40,8 @@ def test_missing_task_list(self):
         self.assertDictEqual(wf_spec.inspect(), expected_errors)
 
     def test_empty_task_list(self):
-        wf_def = """
-            version: '2.0'
-
-            sequential:
-                description: A basic sequential workflow.
-                tasks: {}
-        """
-
```

Comment:
We only put WF definition to `orchestra/tests/fixtures/workflows/mistral` if it will be commonly used.  If the WF definition is only used in only 1 test, we inline the definition in the test for easier references. Also, this change doesn't seem to have anything to do w/ this PR.